tox.ini

```diff
@@ -6,12 +6,18 @@ isolated_build = true
 
 [testenv]
 extras = test
-deps = coverage
+deps =
+  coverage
+  mypy
```

Comment:
You've added `mypy` in the `test` extras already so this is unnecessary.

Reply:
Done